src/components/user-info.js

```diff
@@ -3,21 +3,40 @@ import React from 'react';
 const INFO = 'info';
 
 export default function Info (props) {
-    const { info = {}, onChange } = props;
-    const _onChange               = e => onChange(e, INFO);
+    function errorClass (valid) {
```

Comment:
(minor)
the good practice is to name component the same way as a file name. In this case it would be `UserInfo`.
It's not a strickt rule.

Reply:
It's better to add some className instead of set an empty line